This doesn't build for me.

<details><summary>The build error</summary>

```
-- Could NOT find KF5Wallet (missing: KF5Wallet_DIR)
-- Could NOT find KF5Wallet: found neither KF5WalletConfig.cmake nor kf5wallet-config.cmake
CMake Error at /nix/store/cmriqbxnpcgmbll06pnl1h3va08x2xvr-cmake-3.16.5/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find KF5 (missing: Wallet) (Required is at least version
  "5.62.0")
Call Stack (most recent call first):
  /nix/store/cmriqbxnpcgmbll06pnl1h3va08x2xvr-cmake-3.16.5/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  /nix/store/hxvvgjkvpp22g1rykzcb734649sgzjz0-extra-cmake-modules-5.66.0/share/ECM/find-modules/FindKF5.cmake:110 (find_package_handle_standard_args)
  CMakeLists.txt:38 (find_package)


-- Configuring incomplete, errors occurred!
See also "/build/kwallet-pam-5.17.5/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/znxzdhm9hplgijnwgisj95khf5g88kpd-kwallet-pam-5.17.5.drv' failed with exit code 1
cannot build derivation '/nix/store/shahgnysbssvysmza9fvh5rrb75gg9ih-gdm.pam.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/rg1lr50hdf3i1pjih6phsiqazsyky3vb-kdm.pam.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/18swwxbj8hhdrpsydkk7gaplqnwd6vf4-lightdm.pam.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/0sbrpqyyw86zy3h4d38qy7v0c0axq6qx-sddm.pam.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/vhin09vlcsyqd1cw2s3bm01zd0hjsjd1-system-path.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/mi6dhdjrgl4nzmqigag91p4lspck371p-dbus-1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/w6hk1qksyi43w55647m5wkppdkp8r13z-unit-polkit.service.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/3idhgmb8w406ksn93fg12m2v6kjjbfww-unit-systemd-fsck-.service.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/833injq3r8l3lgx33k9hn01aja76sm51-unit-dbus.service.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/bn5zxj0ndac4bpxaxln8pia969k98clc-unit-dbus.service.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/h9c7zhvw3r6z8y5g1s9bsc3052yli21y-system-units.drv': 3 dependencies couldn't be built
cannot build derivation '/nix/store/y8dl7l74qc50jb7k828vgy6q8l5rfwyn-user-units.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/iggcv5dx7vs4085hf131lqh72vrbsnv3-etc.drv': 8 dependencies couldn't be built
cannot build derivation '/nix/store/rk4pl3c261hn73rivvcknbi5dl8ghc75-nixos-system-nixos-testing-20.09pre-git.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/ckjsj74a7mdp8jjqcibfn7xxs7gscnv6-closure-info.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/jxlfrs62yaaxv4x210wisiy5jrxnfphn-run-nixos-vm.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/avcbdpw6byq55jw7pkwxs4zx2390xzph-nixos-vm.drv': 2 dependencies couldn't be built
error: build of '/nix/store/avcbdpw6byq55jw7pkwxs4zx2390xzph-nixos-vm.drv' failed
```

</details>

However, putting back the `kwallet` dependency in `buildInputs` helps resolving this, and also fixes the coredump. I'm not sure why it was removed here, but it seems like putting it back and leaving the rest alone resolves the linked issue.

--- a/pkgs/desktops/plasma-5/kwallet-pam.nix
+++ b/pkgs/desktops/plasma-5/kwallet-pam.nix
@@ -7,8 +7,15 @@ mkDerivation {
   postPatch = ''
     sed -i pam_kwallet_init -e "s|socat|${lib.getBin socat}/bin/socat|"
   '';
+
+  # We get a crash when QT_PLUGIN_PATH is more than 1000 characters.
+  # pam_kwallet_init passes its environment to kwalletd5, but
+  # wrapQtApps gives our environment a huge QT_PLUGIN_PATH value. We
+  # are able to unset it here since kwalletd5 will have its own
+  # QT_PLUGIN_PATH.
   postFixup = ''
-    wrapQtApp $out/libexec/pam_kwallet_init
+    wrapProgram $out/libexec/pam_kwallet_init --unset QT_PLUGIN_PATH
   '';
+
   dontWrapQtApps = true;
 }

